apps/storybook.namekit.io/stories/Namekit/Input.stories.tsx

```diff
@@ -112,6 +112,17 @@ export const WithSlotRight: Story = {
   },
 };
 
+export const WithError: Story = {
+  args: {
+    variant: "primary",
+    inputSize: "medium",
+    placeholder: "Say something",
+    slotPosition: "right",
```

Comment:
Let's remove any use of slots from this story. It's best to make our examples more "pure" to show just 1 key idea at a time.